invenio_accounts/ext.py

```diff
@@ -53,6 +53,18 @@ def hash_password(password):
     return current_app.extensions['security'].pwd_context.hash(password)
 
 
+def load_obj_or_import_string(imp):
```

Comment:
This would look better if it got moved to the utils.py file. :)
It would also be a good idea to add both of [these](https://github.com/inveniosoftware/invenio-records-rest/blob/4932d82b922f67b3062ecb5c9ae1cf267e97d4ae/invenio_records_rest/utils.py#L60-L81) in there just in case for future use.

Reply:
This function can then be replaced by `obj_or_import_string`.